src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -260,6 +260,11 @@ class OPENASSETIO_CORE_EXPORT Manager final {
    * @param terms Map of terms to be substituted by the manager.
    *
    * @return Substituted map of terms.
+   *
+   * @throws errors.NotImplementedException when this method is not
+   * implemented by the manager.
```

Comment:
There is the `@see` below, and the enum links back to the method. Could we just move the `@see` to be in the same sentence, perhaps?

Reply:
I think it'd be nice to make it really obvious for people, we've seen before that people don't always follow refs and links and join the dots.

Reply:
Added here, along with adding the note to all the completely missed methods. : https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/3596f290647e066f044c34c6294fe034b4b8a263

[Question] I wonder if there's additional doc we should add to the `Exception` tag overloads now, as they have a `@throws` from a seperate mechanism, it may be less obvious that they do exception conversion from BatchElementErrors(There is a `@see` link that tells you, but it's not in the exceptions section.) We could maybe do `@throw Converted BatchElementException See @ref BatchElementErrorPolicyTag::Exception.` so it's clearer when just looking at the exceptions section of the function documentation.

Reply:
Feel free to add anything that clarifies 👍

Reply:
Resolving for the original omission.